testnets/entrypointtestnet/chain.json

```diff
@@ -45,10 +45,10 @@
     },
     "versions": [
       {
-        "name": "v1.1.1",
-        "recommended_version": "v1.1.1",
+        "name": "v1.2.0",
```

Comment:
Can you please keep v1.1.1, and add v1.2.0 as a new object at the bottom of the `versions` array?

Reply:
Totally missed this message, oops. Should be OK now.